internal/infrastructure/db/repositories/device_definition_repo.go

```diff
@@ -31,6 +31,7 @@ type DeviceDefinitionRepository interface {
 	GetByMakeModelAndYears(ctx context.Context, make string, model string, year int, loadIntegrations bool) (*models.DeviceDefinition, error)
 	GetBySlugAndYear(ctx context.Context, slug string, year int, loadIntegrations bool) (*models.DeviceDefinition, error)
 	GetAll(ctx context.Context) ([]*models.DeviceDefinition, error)
+	GetDeviceMMY(ctx context.Context) ([]*models.DeviceDefinition, error)
```

Comment:
you'll need to regenerate the mocks per go generate command towards top of this file. this should handle some of the linter errors